Updating

PanylykMykhailo opened the pull request:
Developer workspace

mikescallette committed:
Updating

mikescallette committed:
Small update

mikescallette committed:
Update label Header Page

mikescallette committed:
hardcode some parameters in addFile()

mikescallette committed:
Get only name

mikescallette committed:
Add new Model
Add method Delete in Controller
Add method Rename in Controller
Update RequestModel

mikescallette committed:
Add condition open edit file

mikescallette committed:
add new parameter(newNameFile(RenameFile func))

mikescallette committed:
Added new column ImageType (FE)
This сolumn return image depending type file

mikescallette committed:
Added width column

mikescallette committed:
Fixed to call delete method

mikescallette committed:
Removed unnecessary
comment
Rewrited work action buttons

mikescallette committed:
Update communication method with FileService

mikescallette committed:
Update comm method with FileService

mikescallette committed:
Add model for Deserealize body
Add independent method for FE

mikescallette committed:
Add new service

mikescallette committed:
Changes

mikescallette committed:
Added dependent and hotfix FE

OleksiiDerepa commented on FileSortService/Controllers/FileSortController.cs:
```diff
@@ -87,5 +87,51 @@ public JsonResult SaveFile()
                 return new JsonResult("anon.png");
             }
         }
+
```
Please rename file name to FileController.cs

PanylykMykhailo replied:
Done

OleksiiDerepa commented on FileSortService/Controllers/FileSortController.cs:
```diff
@@ -87,5 +87,51 @@ public JsonResult SaveFile()
             }
         }
+
+        [Route("DeleteFile")]
```
you don't need to use this rote like 'DeleteFile' please delete

PanylykMykhailo replied:
Done

mikescallette committed:
Rename Controller FileSortService=>Filecontroller
Separated work link BE
Added part to connect service each other

mikescallette committed:
Some system update

mikescallette committed:
Merge fileservice_work

mikescallette committed:
small changes

mikescallette committed:
Testing gateway
Combine functional method

mikescallette committed:
Write to two-way call between
FileSortService and FileService(GetOnly)

mikescallette committed:
small changes

mikescallette committed:
update

mikescallette committed:
update

mikescallette committed:
Added condition where
check is exist include folder

mikescallette committed:
Add new parameter

mikescallette committed:
Some problem -gitignore

mikescallette committed:
Problem .gitignore

mikescallette committed:
Problem with .gitignore

mikescallette committed:
Cut unnecessary part url

mikescallette committed:
merging

mikescallette committed:
remove AngularProject

mikescallette committed:
small changes

mikescallette committed:
Update

mikescallette committed:
update method GetAllFile() new
parameter pathFolder
Add new parameter in Model which get
path choose folder
Update another method what use GetAllFile()

mikescallette committed:
Update return Object in Service

mikescallette committed:
Update DtoObject

mikescallette committed:
Generalizing the method

mikescallette committed:
Add new WebApi (PhotoService)

mikescallette committed:
Add new WebApi (VideoService)

mikescallette committed:
Delete default controller in new Project

mikescallette committed:
Add unnecessary package

mikescallette committed:
Update applicationUrl

mikescallette committed:
Add Model and create method which get only
photo with folder

mikescallette committed:
Add path to general api in PhotoService

mikescallette committed:
Add some dependency

mikescallette committed:
Add Controller
Add Model
Add communication service with filesort

mikescallette committed:
Update settings

mikescallette committed:
Update type what search (VideoService)

mikescallette committed:
Rewrote SaveFile method

mikescallette committed:
Update return value SaveFile method

mikescallette committed:
Rewrote SaveFile and add him to async save

mikescallette committed:
Add new parameter currentDirectory
(space to do action)

mikescallette committed:
rewrote a Createfile method
because folder and file adding is differences

mikescallette committed:
update method delete

mikescallette committed:
Added check in RenameFile method

mikescallette committed:
Update Model Folder

mikescallette committed:
Connected to database
Did migration
Also created method which insert category and all extension

mikescallette committed:
.

mikescallette committed:
Update model in new class for to work with File
Create new EndPoint

mikescallette committed:
Open Test folder

mikescallette committed:
Added new parameter which description
path in project file

mikescallette committed:
Added new version Save File method

mikescallette committed:
Update all service and added new model which
help to deserealize

mikescallette committed:
Add new Object for migration also add all
method which work only file service

mikescallette committed:
add new class which get all info
about correlation
Add all to migration but don`t them.
Added all to migration but don't did her.
Added logic which save in database info about create file

mikescallette committed:
Added in migration sql script

mikescallette committed:
small changes

mikescallette committed:
Binary update

mikescallette committed:
update

mikescallette committed:
Merge with workspace

mikescallette committed:
Rewrote parsing JSON File algorithm
Added try/catch construction which help when file name is not correct

mikescallette committed:
Created All Migration

mikescallette committed:
Start to work with GetFileV2

mikescallette committed:
.

mikescallette committed:
merge

mikescallette committed:
Rewrote logic which forming script
Refresh .json files
Added setting to serealize object

mikescallette committed:
Update json files
Remove unnecessary
Clean all comment code

mikescallette committed:
add new method which return architecture file in folder
from database

mikescallette committed:
Rewrote logic which search namecategory with Id

mikescallette committed:
Added new method which write delete script

mikescallette committed:
Rollback last migration
Rename Migration Object(Table)
Added new file which will helping paste symbols in line script

mikescallette committed:
Rewrote WriteScript class

mikescallette committed:
Update migration
